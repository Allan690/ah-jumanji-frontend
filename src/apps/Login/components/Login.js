import React from "react";

const Login = () => (
  <div className="login">
    <h3>Login</h3>
    <label htmlFor="email">
      Email
      <input id="email" name="email" type="email" placeholder="Email Address" />
    </label>
    <br />
    <button type="button">Submit</button>
  </div>
);

export default Login;
