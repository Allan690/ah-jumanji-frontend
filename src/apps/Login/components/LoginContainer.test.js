import React from 'react';

import { shallow } from 'enzyme';

import Login from './LoginContainer';

describe('Navigation', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Login debug />);

    expect(component).toMatchSnapshot();
  });
});
