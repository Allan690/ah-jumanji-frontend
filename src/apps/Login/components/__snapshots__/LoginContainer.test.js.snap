// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer should render correctly in "debug" mode 1`] = `
<div
  className="row login-container"
>
  <WelcomeSection />
  <Connect(LoginForm) />
</div>
`;
